examples/device_flow.rs

```diff
@@ -0,0 +1,40 @@
+use std::time::Duration;
+const CLIENT_ID: &'static str = "********************";
```

Comment:
This should be retrieved through `std::env::var` using the variable `GITHUB_CLIENT_ID` rather than using a const.